tictactoe.py

```diff
@@ -50,16 +55,8 @@ def victory(self, player):
             return True
         return False
 
-    def check_(self, player, row, col):
-        row = int(row)
-        col = int(col)
-        return self.grid[row][col] == "X" or self.grid[row][col] == "O"
-
 def who(turn):
-    if turn % 2 == 0:
-        return 2
-    else:
-        return 1
+    return 2 if (turn % 2 == 0) else 1
```

Comment:
Nice use of ternary.

Another option is to simply use 0 to represent player 1 and 1 to represent player2. Then playerTurn = turn % 2 .